Doc/library/ipaddress.rst

```diff
@@ -317,6 +317,32 @@ the :func:`str` and :func:`int` builtin functions::
 Note that IPv6 scoped addresses are converted to integers without scope zone ID.
 
 
+Other formatting options
```

Comment:
I think we should add `__format__` under the list of methods and document this there, similar to how datetime does it. As written, this section doesn't mention `__format__`. Unless you know that `format()` calls `__format__` (and most people don't), then I'm not sure you'd make the connection that these format options are also available with `str.format()` and f-strings.